plugins/lsp/language-servers/pylsp.nix

```
@@ -12,7 +12,11 @@ in
 {
   # All settings are documented here:
   # https://github.com/python-lsp/python-lsp-server/blob/develop/CONFIGURATION.md
-
+  options.plugins.lsp.servers.pylsp.python = mkOption {
```

```suggestion
  options.plugins.lsp.servers.pylsp.pythonPackage = mkOption {
```
This is a clearer name IMO. cc @MattSturgeon

---

Agreed. Defining `options.plugins.lsp.servers.pylsp = {` once would also be an improvement imo, but that's a nit.

---

Not sure how to make the diff be smaller, but both of these are implemented.

---

> Not sure how to make the diff be smaller

Can't be helped due to the auto-formatting.

You can view the diff with whitespace ignored though: https://github.com/nix-community/nixvim/pull/1785/files?w=1